src/app/footer/footer.component.html

```diff
@@ -180,6 +180,20 @@
                 >
               </a>
             </li>
+            <li class="m-0">
+              <a
+                href="https://www.youtube.com/channel/UCFWNYqxQvVLAuZq8rdOSE4g"
+                target="_blank"
+                rel="noopener noreferrer"
+                class="footer-link"
+                fxLayoutGap="0.5rem"
+                data-cy="footer-youtube-link"
+              >
+                <img class="site-icons-tab" src="../../assets/images/dockstore/youtube.svg" alt="small Youtube logo" /><small
+                  >Youtube</small
```

Comment:
Probably no one will notice, I but I think its YouTube (and above in the alt)

Reply:
I can fix that, might as well before it's merged? @denis-yuen